index.html

```diff
@@ -300,6 +175,193 @@ <h3 class="about-myself-connect">LET'S CONNECT</h3>
           </div>
         </div>
       </section>
+
+      <section class="second-card-section hidden" data-popup-id="myPopup-0">
+        <div class="all-components-single">
+          <div class="tonic-content-single">
+            <h1>Tonic</h1>
+            <i class="fas fa-times close-popup" data-remove-popup="0"></i>
+          </div>
+          <div class="tonic-content-canopy">
+            <p>CANOPY</p>
+            <ul class="tonic-list-single">
+              <li>Back End Dev</li>
+              <li>2021</li>
+            </ul>
+          </div>
+          <div class="first-image-single">
+            <img src="./asset/nature.png" alt="" />
+          </div>
+          <p>
+            Lorem Ipsum is simply dummy text of the printing and typesetting
+            industry. Lorem Ipsum has been the industry's standard dummy text
+            ever since the 1500s, when an unknown printer took a galley of type
+            and scrambled it to make a type specimen book. It has survived not
+            only five centuries, but also the leap into electronic typesetting,
+            remaining essent
+          </p>
+          <div class="tonic-courses-single">
+            <ul>
+              <li><a href="#">html</a></li>
+              <li><a href="#">css</a></li>
+              <li><a href="#">javascript</a></li>
+            </ul>
+            <hr class="line" />
+          </div>
+
+          <div class="btn">
+            <button type="submit" class="btn-tonic-single">See live</button>
+            <button type="submit" class="btn-tonic-single">See Code</button>
+          </div>
+        </div>
+        <div class="end-bar">
+          <div class="line">
+            <img src="/line.png" alt="" />
+          </div>
+        </div>
+      </section>
+
+      <section class="second-card-section hidden" data-popup-id="myPopup-1">
+        <div class="all-components">
+          <div class="tonic-content">
+            <div class="title-tonic">
+              <h1>Tonic</h1>
+              <i class="fas fa-times close-popup" data-remove-popup="0"></i>
+            </div>
+            <div class="tonic-content-canopy">
+              <p>CANOPY</p>
+              <ul class="tonic-list">
+                <li>Back End Dev</li>
+                <li>2021</li>
+              </ul>
+            </div>
+            <div class="first-image">
+              <img src="./asset/Snapshoot(1).png" alt="" />
+            </div>
+            <p>
+              Lorem Ipsum is simply dummy text of the printing and typesetting
+              industry. Lorem Ipsum has been the industry's standard dummy text
+              ever since the 1500s, when an unknown printer took a galley of
+              type and scrambled it to make a type specimen book. It has
+              survived not only five centuries, but also the leap into
```

Comment:
Data in the pops ups is meant to come from the `datas` array in the `app.js` it should not be added there manually 🤔. 

- [ ] Kindly make changes to dynamically place the objects in the corresponding projects dynamically.